README.md

```diff
@@ -98,6 +98,20 @@ const channel = await getOrCreateChannel('myChannel')
 channel.broadcast({ ... })
 ```
 
+To prevent the API from being used before it is ready, a global `fdc3Ready` event is provided:
```

Comment:
Sorry @kriswest - my previous comment was meant for everything in the readme, I didn't intend to show how we obtain fdc3 object in it - just some brief usage examples.

To keep the readme front page concise, I think let's cover fdc3 ready in the spec, and in docs, but not here.

Reply:
In the original request you mentioned the 'getting started section' I wasn't sure what that was and interpreted as the readme... Will strip this out, but let me know if there is somewhere else it should be